apps/monitoring/helm-base-versions/helm-base-versions.yaml

```diff
@@ -1,36 +1,53 @@
-apiVersion: batch/v1
-kind: CronJob
+apiVersion: helm.toolkit.fluxcd.io/v2
+kind: HelmRelease
 metadata:
   name: helm-chart-base-versions
   namespace: monitoring
+  labels:
+    app: helm-chart-base-versions
 spec:
-  schedule: "0 9 * * 1-5"
-  concurrencyPolicy: "Forbid"
-  failedJobsHistoryLimit: 3
-  startingDeadlineSeconds: 300
-  jobTemplate:
+  releaseName: helm-chart-base-versions
+  serviceAccountName: monitoring
+  chart:
     spec:
-      backoffLimit: 3
-      template:
-        metadata:
-          labels:
-            azure.workload.identity/use: "true"
-        spec:
-          restartPolicy: Never
-          serviceAccountName: monitoring
-          containers:
-            - name: helm-chart-base-versions
-              image: hmctspublic.azurecr.io/helm-base-versions:prod-0bf5a71-20240814143926Z #{"$imagepolicy": "flux-system:helm-base-versions"}
-              imagePullPolicy: Always
-              env:
-                - name: CLUSTER_NAME
-                  value: "cft-${CLUSTER_FULL_NAME}-aks"
-                - name: SLACK_WEBHOOK
-                  valueFrom:
-                    secretKeyRef:
-                      name: helm-base-versions
-                      key: slack-webhook
-                - name: ARTIFACT_URL
-                  value: "http://source-controller.flux-system.svc.cluster.local."
-                - name: API_SERVER_URL
-                  value: "https://kubernetes.default.svc.cluster.local"
+      chart: job
+      sourceRef:
+        kind: GitRepository
+        name: chart-job
+        namespace: flux-system
+      interval: 1m
+  interval: 5m
+  values:
+    schedule: "0 9 * * 1-5" # interval 9:00am Monday to Friday
+    concurrencyPolicy: "Forbid"
+    failedJobsHistoryLimit: 3
+    startingDeadlineSeconds: 300
+    successfulJobsHistoryLimit: 3
+    backoffLimit: 3
+    restartPolicy: Never
+    runAsRoot: true
+    kind: CronJob
+    image: hmctspublic.azurecr.io/helm-base-versions:prod-0bf5a71-20240814143926Z #{"$imagepolicy": "flux-system:helm-base-versions"}
+    imagePullPolicy: Always
+    environment:
+      CLUSTER_NAME: "cft-${CLUSTER_FULL_NAME}-aks"
+      SLACK_WEBHOOK:
+        valueFrom:
+          secretKeyRef:
+            name: helm-base-versions
+            key: slack-webhook
+      ARTIFACT_URL: "http://source-controller.flux-system.svc.cluster.local."
+      API_SERVER_URL: "https://kubernetes.default.svc.cluster.local"
+    nodeSelector:
+      agentpool: linux
+    tolerations:
+      - key: dedicated
+        effect: NoSchedule
+        operator: Equal
+        value: jobs
+    jobTemplate:
+      spec:
+        template:
+          metadata:
+            labels:
+              azure.workload.identity/use: "true"
```

Comment:
Shouldnt need this if using the chart-job helm chart, it has this as an option/value input and is defaulted to true
https://github.com/hmcts/chart-job/blob/master/job/values.yaml#L35